src/main/java/com/exadel/sandbox/service/impl/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,52 @@
+package com.exadel.sandbox.service.impl;
+
+import com.exadel.sandbox.entities.Employee;
+import com.exadel.sandbox.repository.EmployeeRepository;
+import com.exadel.sandbox.service.EmployeeService;
+import liquibase.pro.packaged.E;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeServiceImpl implements EmployeeService {
+
+    @Autowired
+    EmployeeRepository employeeRepository;
+
+
+    @Override
+    public List<Employee> getEmployees() {
+        return employeeRepository.findAll();
+    }
+
+    @Override
+    public Employee getEmployeeByID(Integer id) {
+        return employeeRepository.getOne(id);
```

Comment:
please use findById and if it did not exist throw exception, please check other places